### ISSUE TITLE
After launching start.ps1, Flanneld is stuck in "Waiting for the Network to be created"

### ISSUE THREAD

**Issue by saunderslink** [MicrosoftDocs/Virtualization-Documentation#1013]
I'm receiving this error when running start.ps1 but the resolution/workaround steps do nothing. Another error that shows

"Error registering network: failed to acquire lease: node "WINDOWS NODE" pod cidr not assigned"

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 8d162332-10db-b9f2-92cc-19475ca448ed
* Version Independent ID: 82e7e978-51d0-e563-87bf-e0c82c2bb04a
* Content: [Troubleshooting Kubernetes](https://docs.microsoft.com/en-us/virtualization/windowscontainers/kubernetes/common-problems#feedback)
* Content Source: [virtualization/windowscontainers/kubernetes/common-problems.md](https://github.com/Microsoft/Virtualization-Documentation/blob/master/virtualization/windowscontainers/kubernetes/common-problems.md)
* Product: **containers**
* GitHub Login: @gkudra-msft
* Microsoft Alias: **gekudray**

**Comment by daschott** [comment]
Which windows OS version is this and which network mode is passed to Windows? 
Can you try to launch flannelD in particular?

Can you share
`kubectl get pods -n kube-system`

To find the flannel pod IP and share:
```
kubectl exec -n kube-system kube-flannel-ds-amd64-<someid>  cat /etc/kube-flannel/net-conf.json
kubectl exec -n kube-system kube-flannel-ds-amd64-<someid>  cat /etc/kube-flannel/cni-conf.json
```

**Comment by daschott** [comment]
@saunderslink is this issue still active?

**Comment by saunderslink** [comment]
@daschott Thanks for checking. This issue is resolved.

My Windows nodes needed the pre-requisite patch.

**Comment by AndreyCher** [comment]
Hi, i have same problem. I configured test infrastructure on vmware workstation, kubernetes cluster are configured with one master node , all configuration maked with officialquides from microsoft and kuberntest. 
But when i trying add win node i see what it have status NotReady.
From windows side when work start.ps1 script i see looped message "Waiting for the Network to be created"
Firewall on windows node disabled
Kubernetes version 1.15.0
Windows server 2019 with all updates
In attachment  file with statuses of kubernetes containers and flannel configuration
[kubernetes.txt](https://github.com/MicrosoftDocs/Virtualization-Documentation/files/3396446/kubernetes.txt)

**Comment by saunderslink** [comment]
I found the resolution to be a patch required on the Windows node KB4489899<https://support.microsoft.com/help/4489899>.

MS updated their documentation so it is more apparent that the nodes require this patch, See the overlay networking section at the top of this document.

https://docs.microsoft.com/en-us/virtualization/windowscontainers/kubernetes/getting-started-kubernetes-windows

Kris
________________________________

**Comment by AndreyCher** [comment]
@saunderslink I am readed this reference, and was tryed install this update, but recieve error "The update is not applicable to your pc", i thinks it happens becouse this update already integrated in my windows server build

**Comment by nessangynn** [comment]
Hello, I have this issue too! and I could not figure out how to stop it.

**Comment by guhuajun** [comment]
@AndreyCher @nessangynn 
Please pay attention to these so-called servicing stack update, and make sure the OS is upgraded to the required version(17763.379, use winver to double check)

**Comment by xie-qianyue** [comment]
@AndreyCher @nessangynn Have you guys solved this problem? I have this problem too!
@guhuajun I have checked my os version, its 17763.1158, which is released at 2020.04.14.
